src/System.Memory/src/System/Buffers/Text/StandardFormat.cs

```diff
@@ -0,0 +1,123 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Buffers.Text
+{
+    /// <summary>
+    /// Represents a standard formatting string without using an actual String. A StandardFormat consists of a character (such as 'G', 'D' or 'X')
+    /// and an optional precision ranging from 0..99, or the special value NoPrecision.
+    /// </summary>
+    public struct StandardFormat : IEquatable<StandardFormat>
+    {
+        /// <summary>
+        /// The maximum valid precision value.
+        /// </summary>
+        public const byte NoPrecision = byte.MaxValue;
+
+        /// <summary>
+        /// Precision values for format that don't use a precision, or for when the precision is to be unspecified.
+        /// </summary>
+        public const byte MaxPrecision = 99;
```

Comment:
Do we really want this to be a const? What if we want to change that in the future? i.e. should be be simply a readonly field?

Reply:
Maybe it doesn't need to exist at all? We usually publish argument range limits in documentation rather than as IL metadata.